injective/chain.json

```diff
@@ -48,14 +45,14 @@
     },
     "versions": [
       {
-        "name": "v1.10",
```

Comment:
Instead of writing over the v1.10 version object, can we instead update the v1.11 object?